src/main/java/com/ibm/cloud/sdk/core/util/DateUtils.java

```diff
@@ -55,61 +65,85 @@
  */
 public class DateUtils {
 
-  // RFC 3339 "full-date" used in formatting and parsing.
```

Comment:
the string constants specifying various format pattern strings were getting in the way more than helping so out they go!